package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc. All rights reserved.
+// Copyright 2023 Unity Technologies. All rights reserved.
```

Comment:
I'm not entirely sure, but I don't think we should change the copyright notice in any files we haven't changed. See: https://opensource.google/documentation/reference/copyright
The copyright is kind of a 'timestamp' of work, so as we haven't changed anything I think it should be left alone

Reply:
According to the [Linux Foundation](https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects):

> Don’t change someone else’s copyright notice without their permission
You should not change or remove someone else’s copyright notice unless they have expressly (in writing) permitted you to do so. This includes third parties’ notices in pre-existing code.

I don't know whether this only applies to full Open Source though?? 😖

Reply:
hmm yeah maybe we should be on the safe side and leave this alone then - i was thinking maybe Jeremy or Julien just forgot to change this but theres the chance they just didnt change it at all